src/firebolt/async_db/cursor.py

```diff
@@ -303,10 +303,12 @@ def _row_set_from_response(
             # Skip parsing floats to properly parse them later
             query_data = response.json(parse_float=str)
             rowcount = int(query_data["rows"])
-            descriptions = [
+            descriptions: Optional[List[Column]] = [
                 Column(d["name"], parse_type(d["type"]), None, None, None, None, None)
                 for d in query_data["meta"]
             ]
+            if not descriptions:
```

Comment:
Why can't we just have it empty?

Reply:
Ok, pep 249 says it should be None

Reply:
I feel like it's a bit counter-intuitive to receive empty array in description. For example,
A user does an INSERT query, checks cursor.rowcount is correct then checks for some reason `cursor.description` and it's []. The name itself implies singular as well.
I don't feel too strongly about it, but this is also how we had it up until now.